examples/tools/cmake/cmake_toolchain/build_project.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _examples-tools-cmake-toolchain-build-project:
+
+CMakeToolchain: Building your project
```

Comment:
I think that maybe this should not be an example, this overlaps a bit with this part of the tutorial: https://docs.conan.io/en/2.0-alpha/tutorial/consuming_packages/the_flexibility_of_conanfile_py.html#use-the-layout-method where we explain how the layout() works and that you can use the pre-defined cmake_layout(). Maybe it's just a matter of adding a couple more sentences there, instead of adding a whole example.